beater/beater_test.go

```diff
@@ -0,0 +1,162 @@
+package beater
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	// make sure processors are loaded
+	_ "github.com/elastic/apm-server/processor/error"
```

Comment:
Best load the `_ "github.com/elastic/apm-server/include` package here instead so all processor are always loaded.

Reply:
Good point